web_src/css/shared/settings.css

```diff
@@ -0,0 +1,20 @@
+details.toggleable-item summary {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+/* for specificity to outweigh the rule in base.css */
+.ui.menu details.toggleable-item {
+    user-select: none;
+}
+
+details.toggleable-item summary::after {
+    transition: 0.2s;
+    content: '\203A'; /* chevron right */
```

Comment:
Would be nice if this uses [SVG](https://primer.github.io/octicons/chevron-right-16). It may be possible to inline this as a SVG datauri to avoid additional markup, but it needs to be tested if color is alright with that approach.

Reply:
Thanks.